website/docs/docs/cloud/configure-cloud-cli.md

```diff
@@ -94,9 +94,11 @@ To set environment variables in the dbt Cloud CLI for your dbt project:
 
 ## Use the dbt Cloud CLI
 
-- The dbt Cloud CLI uses the same set of [dbt commands](/reference/dbt-commands) and [MetricFlow commands](/docs/build/metricflow-commands) as dbt Core to execute the commands you provide. For example, use the [`dbt environment`](/reference/commands/dbt-environment) command to view your dbt Cloud configuration details.
-- It allows you to automatically defer build artifacts to your Cloud project's production environment.
-- It also supports [project dependencies](/docs/collaborate/govern/project-dependencies), which allows you to depend on another project using the metadata service in dbt Cloud. 
+The dbt Cloud CLI uses the same set of [dbt commands](/reference/dbt-commands) and [MetricFlow commands](/docs/build/metricflow-commands) as dbt Core to execute the commands you provide. For example, use the [`dbt environment`](/reference/commands/dbt-environment) command to view your dbt Cloud configuration details. With the dbt Cloud CLI, you can:
+
+- Run [multiple invocations in parallel](/reference/dbt-commands#execute-commands) and ensure [safe parallelism](/reference/dbt-commands#parallel-execution), which is currently not guaranteed by `dbt-core`.
```

Comment:
do we use `dbt-core` or should we use dbt Core?

Reply:
I prefer `dbt-core` here, since it is a limitation within that specific Python package. I tend to use "dbt Core" more expansively to refer to dbt OSS (including adapters, sometimes also packages).

Reply:
bloody hell you're fast @jtcohen6 🤣! ok thanks for clarifying that - makes complete sense!